src/command.c

```diff
@@ -72,6 +73,7 @@ const struct command_s cmd_list[] = {
 	{"help", (cmd_handler)cmd_help, "Display help for monitor commands"},
 	{"jtag_scan", (cmd_handler)cmd_jtag_scan, "Scan JTAG chain for devices" },
 	{"swdp_scan", (cmd_handler)cmd_swdp_scan, "Scan SW-DP for devices" },
+	{"probe", (cmd_handler)cmd_probe, "Probe for devices (enable|disable)" },
```

Comment:
This command - does it actually turn probing on and off? It looks like no..?
Also, it appears to only be for Cortex-M, so please word it as such.

Confusing wording for what this actually does